### PR TITLE
#510, #515, #516, #518: Update translations, Fix feedback text input, Remove map rotation

### PR DESCRIPTION
**Pull request by radoslavzeman**

### CODE REVIEW

File: components/map/Map.tsx
Comment (PavelFilo) [inline components/map/Map.tsx:1]: remove everything that was used for rotation... cameraHeading state, mapHeading state, rotateToNorth and also check if everything inside useCameraChangeHandler is used (remove if not)

**Review by PavelFilo — CHANGES_REQUESTED** [review]

File: components/map/Map.tsx
Comment (radoslavzeman) [reply components/map/Map.tsx:1]: Thank you for the comment, remaining old code has been removed :) ✅

File: components/map/CompassButton.tsx [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-import { useMapStoreContext } from '@/state/MapStoreProvider/useMapStoreContext'
-import { cn } from '@/utils/cn'
-
-type Props = {
-  heading: number
-}
-
-const CompassButton = forwardRef<View, Props>(({ heading }, ref) => {
-  const { t } = useTranslation()
-  const { rotateToNorth } = useMapStoreContext()
```
[the remark this answers is not in the document]
Comment (radoslavzeman) [reply components/map/CompassButton.tsx:15]: ✅

**Review by PavelFilo — APPROVED** [review]
good job

**Pull request merged**